core-java-modules/pom.xml

```diff
@@ -31,6 +31,7 @@
         <module>core-java-annotations</module>
         <module>core-java-arrays</module>
         <module>core-java-arrays-2</module>
+		<module>core-java-arrays-3</module>
```

Comment:
Formatting is also off here.

Reply:
I applied the formatter and now I see that all the file looks like being modified. Let me know if it is ok now.

Reply:
Ok, I think now it is correct.